app/assets/javascripts/fe/fe.public.js

```diff
@@ -105,6 +105,7 @@
 			$('#page_ajax_spinner').hide();
 			$('.reference_send_invite').button();
 			updateTotals();
+			$(document).trigger('fePageLoaded'); // allow other code to handle page load event by using $(document).on('pageLoaded', function() { ... });
```

Comment:
Ah, thanks.  I was trying to use $.fe.trigger('pageLoaded') but it seems to only work on elements, and $(document).trigger('...') seems to be standard as it's in an example at the end of https://learn.jquery.com/events/introduction-to-custom-events/